client/src/App.tsx

```diff
@@ -27,7 +28,8 @@ function App(): JSX.Element {
                   <Route exact path="/login" component={Login} />
                   <Route exact path="/signup" component={Signup} />
                   <ProtectedRoute exact path="/listings" component={Listings} />
-                  <ProtectedRoute exact path="/user/profile" component={UserProfile} />
+                  <ProtectedRoute exact path="/user/editprofile" component={EditProfile} />
```

Comment:
I suggest following a naming convention for the routes so camelCase `/user/editProfile` 

same with the following route

Reply:
I was not aware that we could use camelCase for route naming, I have updated as suggested. Thanks!